backend/llm/openai_functions.py

```diff
@@ -27,17 +46,22 @@ def __init__(
         max_tokens: int,
         brain_id: str,
         user_openai_api_key: str,
-    ) -> None:
-        # Call the constructor of the parent class (BrainPicking)
+        # TODO: add streaming
+    ):
         super().__init__(
             model=model,
             chat_id=chat_id,
             max_tokens=max_tokens,
             user_openai_api_key=user_openai_api_key,
             temperature=temperature,
             brain_id=str(brain_id),
+            streaming=False,
```

Comment:
why ?

Reply:
not implemented yet. That was the discussion this morning